dags/analyzer_helper/tests/unit/test_fetch_discord_platforms.py

```diff
@@ -0,0 +1,85 @@
+import unittest
+from unittest.mock import patch
+
+from dags.analyzer_helper.discord.fetch_discord_platforms import FetchDiscordPlatforms
+
+
+class TestFetchDiscordPlatformsUnit(unittest.TestCase):
+    @patch('fetch_discord_platforms.MongoClient')
+    def test_fetch_all(self, MockMongoClient):
+        mock_client = MockMongoClient.return_value
+        mock_db = mock_client['Core']
+        mock_collection = mock_db['platforms']
+
+        sample_data = [
```

Comment:
Please update the sample data using the sample here we gave [link](https://rndadocs.notion.site/Analyzer-Discord-Pipeline-5009512d82a243deb35b3388e17b29c5#:~:text=A%20sample%20platform%20for%20discord%20on%20Core.platforms%20%20collection)